helix-core/src/main/java/org/apache/helix/controller/stages/CompatibilityCheckStage.java

```diff
@@ -55,7 +55,7 @@ public void process(ClusterEvent event) throws Exception {
             "incompatible participant. pipeline will not continue. " + "controller: "
                 + manager.getInstanceName() + ", controllerVersion: " + properties.getVersion()
                 + ", minimumSupportedParticipantVersion: "
-                + properties.getProperty("miminum_supported_version.participant")
+                + properties.getProperty("minimum_supported_version.participant")
```

Comment:
What's the difference here?

Reply:
The previous one has a typo.

Reply:
And it caused the log to always show "null" for minimum support version, as it cannot find it in the property file. So the log is very confusing during debug.